CMakeLists.txt

```diff
@@ -16,14 +16,15 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/CMakeModules/Modules)
 ###############################################################################
 
 # Valid applications and choices
-list(APPEND VALID_APPS ATM ATMW S2S S2SW NG-GODAS NG-GODAS-NEMSDATM)
+list(APPEND VALID_APPS ATM ATMAERO ATMW S2S S2SW NG-GODAS NG-GODAS-NEMSDATM)
 set(APP NONE CACHE BOOL "Application Name")
 if(NOT (APP IN_LIST VALID_APPS))
   message(FATAL_ERROR "${APP} is not a valid application.\nValid Applications are: ${VALID_APPS}")
 endif()
 
 set(FMS             OFF CACHE BOOL "Enable FMS")
 set(FV3             OFF CACHE BOOL "Enable FV3")
+set(UFS_GOCART      OFF CACHE BOOL "Enable GOCART")
```

Comment:
Use `GOCART` instead of `UFS_GOCART`, similar to all other options which correspond to subdirectory (submodule) names. Build option to turn on MOM6 is called `MOM6` not `UFS_MOM6`.

Reply:
We had agreed on `UFS_GOCART` last week.

Reply:
We agreed that the top-level cmake build option that enables GOCART submodule will be `UFS_GOCART`?